docs/meshstack.gcp.index.md

```diff
@@ -234,3 +234,28 @@ We recommend that configuration include at least 3 characters of the random para
       "%.15s-%.10s-%.3s"
 }
 ```
+
+## Audit Logs for meshfed-service User 
+
+The actions of the `meshfed-service` User can be monitored via [Audit Logs](https://cloud.google.com/logging/docs/audit/). This allows an in-depth view meshStack activities for GCP project at any moment.
+
+### Enable Audit Logs
+
+> Enabling Audit Logs may incur charges.
+
+meshcloud recommends to enable Audit Logs on the organizational level for monitoring `meshfed-service` User. This is achivied by following these steps:
+
+1. Navigate to the organizational level in [GCP Cloud Console](https://console.cloud.google.com/)
+2. Navigate to [IAM & Admin --> Audit logs](https://console.cloud.google.com/iam-admin/audit)
+3. Filter the table for `Cloud Resource Manager API` and select the resulting entry
+4. Enable all log types
+
+You may want to check the [official Google instructions](https://cloud.google.com/logging/docs/audit/configure-data-access#config-console-enable) on enabling Auit Logs for further information.
+
+The below screen shot show how to set up the Audit Logs for the organization `dev.meshcloud.io`
```

Comment:
The screen shot below shows how to set up the Audit Logs via the Google Cloud Console